### ISSUE TITLE
dose_test failure: syslog_output about Boost.Asio latency

### ISSUE THREAD

**Issue by DonOregano** [SafirSDK/safir-sdk-core#396]
These happen quite often in the new Jenkins setup, in both standalone and multinode configurations. Probably due to the VM system being rather overloaded.

```
syslog is not empty:
"<12>Jul 18 01:15:03 core-debian-bullseye-64-test-slave safir_control[3997524]: (0) Boost.Asio latency for 'SpRawHandler' is at 2736 milliseconds. If this happens a lot your system is overloaded and may start misbehaving.
"
```

**Comment by DonOregano** [comment]
Closing under a deliberate tracking-policy change, not because this is resolved — it still happens.

**Known test flakiness now lives in [TEST_STATUS.md](https://github.com/SafirSDK/safir-sdk-core/blob/develop/TEST_STATUS.md) only, not in issues.** An open issue is a claim that someone intends to do the work; a flake entry is a statement of a known condition. Keeping both in parallel never stays in sync — this issue is the proof, having sat open for four years while the actual triage state accumulated elsewhere. From now on an issue gets raised when we decide to investigate or fix a specific flake.

Where this one is recorded:
- **`syslog_output` is a canary — read its body** — the check simply fails if syslog is non-empty (`run_dose_tests.py.in:594`, no filtering), so it is a detector rather than one bug. Always read the `<failure>` body; it often explains a co-occurring failure in the same run.
- **`safir_control.0.returncode` + `syslog_output` (correlated; overload-triggered)** — the full chain as captured on CI #132: runner stall (`Boost.Asio latency for 'SpRawHandler' is at 5583 ms … 7902 ms`, exactly the symptom reported here) → lost UDP acks → `Excessive retransmits … excluding it!` → `safir_control` exiting 1.

Still occurring on GitHub Actions: runs 32461278855 (2026-08-21), 32143455257 (2026-08-18), 28461731183 and 28340817399 (2026-06). The trigger remains what was suspected here in 2022 — an overloaded runner rather than a product fault.

One thing did come out of it that is *not* flakiness: the overload path ends in `Coordinator.h:781` throwing when a `SystemStateMessage` lists a node as both alive and dead, which aborts the whole node instead of reconciling. That is a real robustness gap and will get its own issue if we decide to act on it.
